### DISCUSSION TITLE
how to use conditions

### DISCUSSION THREAD

**Discussion by benkya** [AutomaApp/automa#368] [Q&A]
hi,
freshman here, I want to know how can use conditions , yes , means, if some element exist, to do something, else, do other.Are there any examples for this？ 
TKKKKKKKKKKKKKKS

**Comment by Kholid060** [comment]
You can use the [element exists block](https://docs.automa.site/blocks/element-exists.html) to check if an element exists or not. For example,

![image](https://user-images.githubusercontent.com/22908993/155045678-133e19e0-82c4-4044-a631-81dae13d9abf.png)

If the element exists, it will execute the get text block otherwise it executes the new tab block.

**Reply by benkya** [reply]
Thanks a lot, but there is another question, I set the function as your answer ,  but when I run it, both 'Get text' and 'New tab'  running. Any other setting here?
